logging_utilities/formatters/json_formatter.py

```diff
@@ -171,11 +175,17 @@ def _add_object_to_message(self, record, obj, message):
             elif isinstance(value, str):
                 # When the value is a string it can contain formatting strings (e.g. "%(asctime)s")
                 # therefore try to format it.
-                self._style._fmt = value  # pylint: disable=protected-access
-                message[key] = super().formatMessage(record)
+                style = self._style_constructor(value)
+                message[key] = style.format(record)
                 if self.remove_empty and message[key] == '':
                     del message[key]
 
+    def usesTime(self):
```

Comment:
Not sure if this will work as expected, when called in line 196.
I think self.usesTime() will be `-1`, when no asctime is found in the dict, not `None`

Reply:
Hey good catch

Reply:
I'll changed line 76 to 
```python
self._use_time = str(fmt).find('asctime') >= 0
```